Log-Correlation-RefApps/flask-app/values.yaml

```diff
@@ -0,0 +1,88 @@
+global:
+  sfappname: Expenses
+  sfprojectname: Banking
+  sfappname_key: snappyflow/appname
+  sfprojectname_key: snappyflow/projectname
+  key: aFzYgxssaIu0yyeQvJt/GxrydJq4J2drMJcP4rCYSzp2m92nzD6jGsw5OVqjGhY7XKVdrwN30j/LZ/8i1A9rdDtyvgInaF32++RqseERSUdM1M2A97S4NHXVA6YGSZ9gFU4yVpcXzKoKtR0hyh9sZmRvnxiqUeajr1CktZ5hsLhNMNEIguQLQ1IUN/qAHJSy2pZh50Tiv8xUueo0J4spE4K04YFRJcp/zLBRcArxDGlPA3hl7E3BNMtZ7yFXYheR61YQN5ryrErxsJEenpB4knoVWfiJ7xYfr1C9FoAlyMSujKHJZcxS8x8PcXCsh74p0rRA0OJgfFS/fmJrMDBy65EXZ47T+oPguEuJx+Nn0FZwDqYvmGFCtmEKqsgClZxgACzRhaTgAEsIPZ+Wi5GrnVuou1cCF+uULxdXm2qClTmeauKgnTb4+MhwRzGbdFcTglxXhJkWAaCsboqrcthoCSYQXdaSX5XAdUNLElCZ9UIZEH3JFPsBMOWRLXI3sjK5Xd1CPOQSt6QsnAR8+m0JfPrYljHOqB4kXGZLxJytA7uoRunYIM06YtVrpTV+91djs/lTrO9/Vi5mzHcX/q/AA6TKuzA3sF7JPuo7TFVN7ppKfQ3tOPtwBgNg8GxjLNaGZqOLvmde8hv5a3FcDJ7AWfcJp5cgDXlGJ2Uxx8tKdXd1uXg8qjJduispmvBwwFCHcO53ngeI0jKmRoR32oBPEoQ566K3KdWE8FEDsofBBAnVxZk9KNicUVL8peg22U78nbG4yEnw/35GmB80+C2NbQJI1X4RitYzbzL1XDeOtiRvJ+vkuF08Hp9kIezcG974Vj++MTaTqs314WCOYv/TksYFQ4jTJUaGfUdd8d4P5M++kNvKin7IPsCmPObFyEtC0tK8Zr0EHwDV49qBCPz8cQ==
```

Comment:
Remove this profile key

Reply:
Hi @Pradeep-Jaiswar-ML , removed the profile key.